tests/fixtures/1.1/release-packages/ocds-213czf-000-00001.json

```diff
@@ -0,0 +1,1143 @@
+{
```

Comment:
The name of this file should not be "0004-package" or something like that?

Reply:
I don't think so